src/docs/contributing/approach/sdk-docs/write-performance.mdx

```diff
@@ -31,15 +31,21 @@ This file is `index.mdx`. It explains how to install or set up (if appropriate)
 
 To develop content for this file:
 
-1. Determine if the SDK needs an installation step or a property set to enable tracing. If so, add the SDK to `Supported` list in the `<PlatformSection>`, then add a code sample to `performance/enable-tracing/`. If not, add the SDK to `notSupported` list, which will prevent the **Enable Tracing** heading from displaying for this SDK.
+1. If the SDK is still a work-in-progress and is exposed as an experimental or an early access feature, add the platform to the `<PlatformSection>` wrapping the appropriate alert located at the beginning of the content for the page.
 
-2. Add a code sample to `/src/includes/performance/configure-sample-rate/` that covers how to set sample rates.
+1. If the SDK is expected to be used in a high-throughput environment, add the platform to the following `<PlatformSection>` which advises testing before deploying to production.
```

Comment:
So this I'm pretty sure this 1. trick works with markdown, but I'm not sure if readers are going to be viewing these through a markdown reader, or if they're just going to be reading the raw contents. If it's the latter, then this may look a little silly and would be better off left as a bullet list. If not, then I think this makes adding in new items easier. Would love some thoughts here.

Reply:
We use the 1. method, so all good.